sites/fast-website/src/public/index.html

```diff
@@ -1,13 +1,13 @@
 <!DOCTYPE html>
-<html>
+<html lang="en">
```

Comment:
can you also add `dir="ltr"`? I think we have a few components that look for that